<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Очередь:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget1"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spinBoxQ1"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushBackButton">
          <property name="text">
           <string>PushBack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushFrontButton">
          <property name="text">
           <string>PushFront</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="popFrontButton">
          <property name="text">
           <string>PopFront</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="popBackButton">
          <property name="text">
           <string>PopBack</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Удаленный элемент:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDeletedItem">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="randomButton1">
        <property name="text">
         <string>Заполнить случайными элементами</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Очередь пуста:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label11">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Размер очереди:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label12">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Первый элемент:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label13">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Последний элемент:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label14">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="buttonTask">
        <property name="text">
         <string>Удалить из
очереди все элементы, находящиеся между максимальным и
минимальным.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
